civihr_default_theme/assets/js/civihr_default_theme.script.js

```diff
@@ -296,4 +296,8 @@
     $('#' + this.id + '-' + 'day').val(dateValues.date).trigger('change');
     $('#' + this.id + '-' + 'year').val(dateValues.year).trigger('change');
   }
+
+  Drupal.civihr_theme.deleteEmergencyContact =  function(id) {
+    CRM.api3('contact', 'deleteemergencycontact', {'id':id});
+  }
```

Comment:
Did you ran the semistandard checks before the commit, the format seems not correct. Should be as following:
```js
Drupal.civihr_theme.deleteEmergencyContact =  function (id) {
   CRM.api3('contact', 'deleteemergencycontact', { 'id': id });
}
```

Reply:
It was fixed as part of next commit.